src/caip-types.ts

```diff
@@ -74,7 +104,9 @@ export const CaipAssetIdStruct = pattern(
   string(),
   CAIP_ASSET_ID_REGEX,
 ) as Struct<CaipAssetId, null>;
-export type CaipAssetId = `${string}:${string}/${string}:${string}/${string}`;
+export type CaipAssetId =
+  | `${string}:${string}/${string}:${string}`
```

Comment:
According to the spec, an `asset_id` must include the `"/" + token_id` suffix.

Reply:
Actually it specifies that `token_id` is optional: "The optional addition of an asset ID suffix separated by / uniquely identifies an addressible asset of a given type in a developer-friendly fashion"

And in its examples it demonstrates this (many chains do not use the `token_id` at all):
```
# Ether Token
eip155:1/slip44:60

# DAI Token
eip155:1/erc20:0x6b175474e89094c44da98b954eedeac495271d0f

# Solana Mainnet USDC
solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp/token:EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v
```

Reply:
Actually, now I've re-read the spec for the millionth time, and I can see that it was written in a way that can be interpreted in two different ways. I see now that your interpretation may be more valid than mine. I'll change this PR to reflect it.